conbench/tests/entities/test_commit.py

```diff
@@ -135,9 +147,15 @@ def test_get_github_commit_and_fork_point_sha_pull_request(branch, pr_number):
         "branch": "dianaclarke:ARROW-13266",
         "fork_point_sha": "780e95c512d63bbea1e040af0eb44a0bf63c4d72",
     }
-    assert (
-        get_github_commit(repo, branch=branch, sha=sha, pr_number=pr_number) == expected
-    )
+    try:
+        result = get_github_commit(repo, branch=branch, sha=sha, pr_number=None)
```

Comment:
Yes, here: `pr_number=pr_number` turned to be `pr_number=None` :fire: -- Fixing this :fire_engine: